File: App_19k.Service/Implementacion/RespuestaService.cs
using System;
using System.Collections.Generic;
using System.Text;
using App_19k.Domain;
using App_19k.Repository;
using App_19k.Repository.Implementacion;
using App_19k.Repository.ViewModel;

namespace App_19k.Service.Implementacion
{
    public class RespuestaService : IRespuestaService
    {
        private IRespuestaRepository RespuestaRepository;

        public RespuestaService(IRespuestaRepository RespuestaRepository)
        {
            this.RespuestaRepository = RespuestaRepository;
        }

        public bool Actualizar(RespuestaViewModel respuesta)
        {
            return RespuestaRepository.Actualizar(respuesta);
        }

        public bool Delete(int id)
        {
            throw new NotImplementedException();
        }

        public bool Enviar(RespuestaViewModel respuesta)
        {
            return RespuestaRepository.Enviar(respuesta);
        }

        public Respuesta Get(int id)
        {
            throw new NotImplementedException();
        }

        public IEnumerable<Respuesta> GetAll()
        {
            throw new NotImplementedException();
        }

        public Respuesta ObtenerRpta(int idProyecto)
        {
            return RespuestaRepository.ObtenerRpta(idProyecto);
        }

        public bool Save(Respuesta entity)
        {
            throw new NotImplementedException();
        }

        public bool Update(Respuesta entity)
        {
            throw new NotImplementedException();
        }
    }
}

File: src/main/java/com/pe/app19k/service/impl/RespuestaService.java
package com.pe.app19k.service.impl;

import com.pe.app19k.entity.Respuesta;
import com.pe.app19k.repository.IRespuestaRepository;
import com.pe.app19k.service.IRespuestaService;
import org.springframework.beans.factory.annotation.Autowired;
import org.springframework.stereotype.Service;
import org.springframework.transaction.annotation.Transactional;

import java.util.List;
import java.util.Optional;

@Service
public class RespuestaService implements IRespuestaService {
    @Autowired
    private IRespuestaRepository respuestaRepository;
    @Override
    public Respuesta Insert(Respuesta respuesta) throws Exception {
        return respuestaRepository.save(respuesta);
    }

    @Override
    public Respuesta Update(Respuesta respuesta) throws Exception {
        return respuestaRepository.save(respuesta);
    }
    @Transactional(readOnly=true)
    @Override
    public List<Respuesta> FindAll() throws Exception {
        return respuestaRepository.findAll();
    }
    @Transactional(readOnly=true)
    @Override
    public Optional<Respuesta> findById(Integer integer) throws Exception {
        return respuestaRepository.findById(integer);
    }

    @Override
    public void deleteById(Integer integer) throws Exception {
        respuestaRepository.deleteById(integer);
    }

	@Override
	public List<Respuesta> fetchByProjectId(Integer id) throws Exception {
		return respuestaRepository.fetchByProjectId(id);
	}
}
